Dockerfile

```diff
@@ -17,12 +17,12 @@ COPY pkg/ pkg/
 COPY vendor* vendor/
 
 
-FROM builder as vendor-Enabled
+FROM builder as vendor-enabled
 ARG TARGETARCH
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=${TARGETARCH} go build -mod=vendor -a -o manager main.go
 
-FROM builder as vendor-Disabled
+FROM builder as vendor-disabled
```

Comment:
[These](https://github.com/RocksLabs/kvrocks-operator/blob/unstable/Makefile#L132) should also be modified

Reply:
done